TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift

```diff
@@ -25,26 +25,25 @@ public protocol FactorPayload {
   var friendlyName: String { get }
   ///The unique SID identifier of the Service.
   var serviceSid: String { get }
-  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
-  ///because the systems that will process this attribute assume it is not directly identifying information.
+  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information) because the systems that will process this attribute assume it is not directly identifying information.
   var identity: String { get }
   ///Type of the factor.
   var factorType: FactorType { get }
 }
 
-///Describes the information required to create a **Factor** which type is **Push**.
+/// Describes the information required to create a **Factor** which type is **Push**.
 public struct PushFactorPayload: FactorPayload {
-  ///A human readable description of this resource, up to 64 characters. For a push factor, this can be the device's name.
+  /// A human readable description of this resource, up to 64 characters. For a push factor, this can be the device's name.
   public let friendlyName: String
-  ///The unique SID identifier of the Service.
+  /// The unique SID identifier of the Service.
   public let serviceSid: String
-  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
+  /// Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
   ///because the systems that will process this attribute assume it is not directly identifying information.
   public let identity: String
   ///Type of the factor, push by default.
   public let factorType: FactorType = .push
-  ///Registration token generated by APNS when registering for remote notifications.
-  public let pushToken: String
+  ///(Optional) Registration token generated by APNS when registering for remote notifications. Sending nil or empty will disable sending push notifications for challenges associated to this factor.
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/012ddda094c6c1e8c6d0425188ef9a322b4ad2d7/TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift#L45"></span>Line should be 195 characters or less: currently 198 characters<br />
<code>line_length</code> <code>FactorPayload.swift:45</code></td>
    </tr>
  </tbody>
</table>